perf/benchmark/run_benchmark_job.sh

```diff
@@ -40,6 +40,7 @@ export PILOT_CLUSTER="${PILOT_CLUSTER:-}"
 export USE_MASON_RESOURCE="${USE_MASON_RESOURCE:-True}"
 export CLEAN_CLUSTERS="${CLEAN_CLUSTERS:-True}"
 export NAMESPACE=${NAMESPACE:-'twopods-istio'}
+export PROMETHEUS_NAMESPACE=${NAMESPACE:-'istio-system'}
```

Comment:
This means usually Prometheus is in twopods namespace ?

Reply:
good catch, fixed it